Rocket.Chat/cs.lproj/Localizable.strings

```diff
@@ -106,13 +106,15 @@
 "chat.dm.start_conversation" = "Toto je začátek vaší konverzace s %@";
 "chat.channel.start_conversation" = "Toto je začátek konverzace %@";
 "chat.read_only" = "Tato místnost je pouze pro čtení";
-"chat.muted" = "You have been muted"; //TODO
+"chat.muted" = "You have been muted";
+"chat.edit_mode" = "Upravit zprávu. Blízko zrušit.";
```

Comment:
Could you add the strings to the same places in all the languages file? This makes easier to maintain in the future.